java-security/README.md

```diff
@@ -271,22 +271,42 @@ This validator is not part of the default `CombiningValidator`, it needs to be a
 It can be done in the following manner:
 ```java
 JwtValidatorBuilder.getInstance(oAuth2ServiceConfiguration)
-    .with(new JwtX5tValidator(oAuth2ServiceConfiguration))
-    .build();
+		.with(new JwtX5tValidator(oAuth2ServiceConfiguration))
+		.build();
+```
+
+Or it can be used as a standalone `Validator`, by creating a new instance of it and
+calling `JwtX5tValidator.validate(Token token)` method with the token to be validated as a method's parameter.
+See [here](#retrieve-additional-information-from-token) how to get a token from `SecurityContext`
+
+```java
+JwtX5tValidator validator=new JwtX5tValidator(oAuth2ServiceConfiguration);
+		ValidationResult result=validator.validate(token);
 ```
-Or it can be used as a standalone `Validator`, by creating a new instance of it and calling `JwtX5tValidator.validate(Token token)` method with the token to be validated as a method's parameter. See [here](#retrieve-additional-information-from-token) how to get a token from `SecurityContext`
+
+#### Proof Token validation
+
+Once enabled, it will forward the X509 client certificate from the request header `x-fowarded-client-cert`
```

Comment:
Once available, let us add a link to a documentation that describes this feature in detail. This description here is rather technical but for now, it is the best we can do.